experimental/framework/frs-client/package.json

```diff
@@ -44,15 +49,21 @@
     "debug": "^4.1.1"
   },
   "devDependencies": {
+    "@fluid-experimental/fluid-framework": "^0.43.0",
     "@fluidframework/build-common": "^0.22.0",
     "@fluidframework/eslint-config-fluid": "^0.23.0",
     "@microsoft/api-extractor": "^7.16.1",
+    "@types/assert": "^1.5.2",
     "@types/mocha": "^8.2.2",
+    "assert": "^2.0.0",
```

Comment:
looks like we don't need `assert` and `types/assert` if it's node-tests only - curtis removed them everywhere else in another pr (browser code should be using the `common-utils` implementation)

Reply:
removed